tpch/execute.py

```diff
@@ -40,7 +40,7 @@
     "dask": lambda x: x.compute(),
 }
 
-DUCKDB_SKIPS = ["q14", "q15"]
+DUCKDB_SKIPS = ["q15"]
```

Comment:
excluding `lit` in `maybe_evaluate` and others would mean not being able to execute q14

we don't have window expressions in duckdb anyway, so for now I've avoided that condition

I suspect that q14 might be similarly problematic for PySpark though, so we may need to find another solution